### PR TITLE
Create CODEOWNERS, requiring review from either Jack or Alex

### PR DESCRIPTION
**Pull request by acmcarther**
This needs a corresponding setting to be enabled in `Settings`, which I'll toggle after this is submitted.

### CODE REVIEW

**Comment by acmcarther** [comment]
Relevant documentation: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners

**Review by jalexanderqed — APPROVED** [review]

**Review by meagancheese — APPROVED** [review]

**Comment by acmcarther** [comment]
Toggled the relevant setting: [Settings] -> [Branches] -> `master` -> `Require review from Code Owners`.

**Pull request merged**
